src/util/styles/fonts.ts

```diff
@@ -5,26 +5,6 @@ import { IRawStyle, IFontWeight } from '@uifabric/styling';
 
 export type Sizes = { [size in TextSize]: string };
 
-export const fontSizes: Sizes = {
-  xSmall: '1.0rem',
-  small: '1.2rem',
-  mediumSub: '1.4rem',
-  medium: '1.5rem',
-  large: '1.8rem',
-  xLarge: '2.2rem',
-  xxLarge: '2.4rem',
-};
-
-export const lineHeights: Sizes = {
-  xSmall: '1.6rem',
-  small: '1.6rem',
-  mediumSub: '2.0rem',
-  medium: '2.0rem',
-  large: '2.4rem',
-  xLarge: '2.8rem',
-  xxLarge: '2.8rem',
-};
-
```

Comment:
Since we're always getting these from the `theme` we don't need them here anymore.